doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -174,6 +174,7 @@ Bluetooth® LE
 -------------
 
 * Updated the Bluetooth LE SoftDevice Controller driver to make the :c:func:`hci_vs_sdc_llpm_mode_set` function return an error if Low Latency Packet Mode (LLPM) is not supported or not enabled in the Bluetooth LE Controller driver configuration (:kconfig:option:`CONFIG_BT_CTLR_SDC_LLPM`).
+* Added a toolkit for distance estimation using channel sounding (:kconfig:option:`CONFIG_BT_CS_DE`).
```

Comment:
Move the entry under Libraries - Bluetooth like this:

* Added the :ref:`cs_de_readme` library.